MODULE.bazel

```diff
@@ -0,0 +1,91 @@
+module(
+    name = "everest-core",
+    version = "0.0.0",
+)
+
+###############################################################################
+# B A Z E L  C E N T R A L  R E G I S T R Y # https://registry.bazel.build/
+###############################################################################
+
+bazel_dep(name = "everest-framework", version = "0.0.0")
+git_override(module_name="everest-framework", commit="aebffb45d9c1dc948ad31e718fc3efe9eac0a9ca", remote="https://github.com/EVerest/everest-framework")
```

Comment:
FIXME: rev @ main

Reply:
main woudn't be reproducible

Reply:
I think I wrote that when it was still pointing to a rev on a feature-branch